pytorch_lightning/loops/fit_loop.py

```diff
@@ -124,6 +124,19 @@ def _results(self) -> ResultCollection:
             return self.epoch_loop.val_loop._results
         raise RuntimeError("`FitLoop._results` property isn't defined. Accessed outside of scope")
 
+    @staticmethod
+    def _is_max_limit_enabled(max_value: Optional[int]) -> bool:
```

Comment:
This will disappear when the suggestion in https://github.com/PyTorchLightning/pytorch-lightning/issues/8818#issuecomment-907825247 is implemented, correct?